internal/featuregate/featuregate.go

```diff
@@ -97,3 +97,9 @@ func (s *Stability) Set(str string) error {
 func (s Stability) Type() string {
 	return "<stability_level>"
 }
+
+// AtLeast will check if the stability level is allowed. For instance experimental would allow public preview, experimental and ga. Whereas ga would only allow ga.
+func (s Stability) AtLeast(stability Stability) bool {
+	return s <= stability
+
+}
```

Comment:
I'm struggling with the name here a little bit, since I'm reading it as the phrase "experimental is at least public preview" which sounds odd to me. 

What about something calling it something like "permits" so then you could read it as "experimental permits public preview functionality"?